posts/2016-09-20-IDA2016-notes.md

```diff
@@ -0,0 +1,126 @@
+# International Data Week 2016 - Meeting Notes and Links
+
+*[International Data Week](internationaldataweek.org) in September 2016 brought together three events, [SciDataCon](http://www.scidatacon.org/), the International Data Forum, and the [Research Data Alliance 8th plenary](https://rd-alliance.org/plenaries/rda-eighth-plenary-meeting-denver-co). Joe Hand from the Dat team attended the conference and collected his notes here.*
+
+## SciDataCon2016
+
+SciDataCon featured sessions with 5-10 quick lightning talk each on a common theme. Notes from some of the more interesting sessions are below.
+
+### [Session on Sustainable Business Models for Data Repositories](http://www.scidatacon.org/2016/sessions/45/)
+
+The first two sessions covered the costs, budgets, and revenue streams for data repositories. Repositories have a wide range of pricing and revenue structures. The witness statements in the session link above provide details on the pricing models. Repositories are using a mix of subscriptions (paid by libraries, institutions, researchers), pay to download (to get specific datasets), or pay to deposit (pay to upload to the repository), and external funding (government, private).
```

Comment:
The last sentence could be: 'These models include: subscriptions paid by libraries... '
also, maybe we can get rid of these parentheses